From 68d8f64267e1611735f9c3187cf91f92ce085a23 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 21 Jul 2011 16:37:22 +0800
Subject: [PATCH 20/43] ENGR00153459 Integrate Huawei MU509 update

Do not build the default ril if the env PREBUILT_3G_MODEM_RIL be true

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 reference-ril/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/reference-ril/Android.mk b/reference-ril/Android.mk
index 0c77559..d53e93c 100644
--- a/reference-ril/Android.mk
+++ b/reference-ril/Android.mk
@@ -2,6 +2,7 @@
 
 # XXX using libutils for simulator build only...
 #
+ifneq ($(PREBUILT_3G_MODEM_RIL),true)
 ifeq ($(BOARD_HAVE_MODEM),true)
 
 LOCAL_PATH:= $(call my-dir)
@@ -54,3 +55,4 @@ else
 endif
 
 endif
+endif
-- 
1.8.0

